homeassistant/components/shelly/manifest.json

```diff
@@ -5,5 +5,6 @@
   "documentation": "https://www.home-assistant.io/integrations/shelly",
   "requirements": ["aioshelly==0.6.2"],
   "zeroconf": [{ "type": "_http._tcp.local.", "name": "shelly*" }],
-  "codeowners": ["@balloob", "@bieniu", "@thecode", "@chemelli74"]
+  "codeowners": ["@balloob", "@bieniu", "@thecode", "@chemelli74"],
+  "quality_scale": "platinum"
```

Comment:
Test coverage isn't complete. That's required for gold.

https://github.com/home-assistant/core/blob/23fae255fff9aac56f760f59c998d5ac51bcb3b6/.coveragerc#L861-L866

Reply:
Thanks, putting this on draft, will get back to it after tests are completed.